bokehjs/test/models/glyphs/rect.coffee

```diff
@@ -297,14 +297,14 @@ describe "Rect", ->
           expect(result3['1d'].indices).to.be.deep.equal([1])
 
         it "should work when axis is log", ->
-          data = {x: [60, 100, 140], y: [60, 100, 140]}
+          data = {x: [1, 10, 100, 1000], y: [1, 10, 100, 1000]}
           glyph_view = create_glyph_view(@glyph, data)
 
           @set_mappers(glyph_view, type="log")
           glyph_view.map_data()
 
-          result4 = glyph_view._hit_point(@geometry4)
-          result5 = glyph_view._hit_point(@geometry5)
+          result4 = glyph_view._hit_point({ vx: 200, vy: 200 })
```

Comment:
This test should probably be replaced with a selenium test. I understand that the intent is to assert that the _hit_point hits the correct glyph at a vx/vy (based on the mapper), but in practice it's difficult to figure out the vx/vy of the glyph. Currently the test passes, but doesn't check much of value.